extensions/amp-subscriptions-google/amp-subscriptions-google.md

```diff
@@ -87,6 +87,37 @@ If `subscribe.google.com` is the "winning" service, the request to the `pingback
 
 Where `data` matches the [entitlements response](https://github.com/subscriptions-project/swg-js/blob/master/docs/entitlements-flow.md#entitlement-response) format.
 
+Additionally, if `subscribe.google.com` returns an entitlement, and `hasAssociatedAccountUrl` and
```

Comment:
The big question here: why is this specific to the `subscribe.google.com`? This seems to belong to the `local` to me. I could be convinced otherwise, but this seems like any service could send their entitlements to local for verification, just as they would for pingback.

Reply:
That makes sense that way of services beyond amp-subscriptions-google come along in the future it will "just work"